File: hugogoApp/src/app/Models/article.ts
export class Article {

    id: number;
    title: string;
    content: string;
    shortDecr: string;
    img: number;
    createdAt: Date;
    updatedAt: Date;

    constructor(id: number, title: string, content: string, shortDecr: string, img: number, createdAt: Date, updatedAt: Date) {
        this.id = id;
        this.title = title;
        this.content = content;
        this.shortDecr = shortDecr;
        this.img = img;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
    }
}

File: Models/Article.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace hugo.Models
{
    public class Article
    {
        public int ID { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public string ShortDecr { get; set; }
        public int Img { get; set;}
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
    }
}
